library/default-synths-extra.scd

```diff
@@ -481,10 +481,10 @@ SynthDef(\superwavemechanics, {|out, pan, freq, sustain, accelerate, detune=0, v
 	var env, sound, i, r, d, bwr, sz;
 	freq = freq * XLine.ar(1, exp(accelerate), sustain);
 	env = EnvGen.ar(Env.sine(1.0), timeScale:sustain, doneAction:2);
-	i = Array.fill(8, {TIRand.kr(0, 3, Impulse.kr(8))});
+	i = Array.fill(8, {TIRand.kr(0, 3, Impulse.kr(sustain/8))});
 	r = [3,5,8,9]/8;
 	detune = min(detune, 1.5);
-	d = Rand(1.6*i - detune, 1.7*i + detune).round(0.25);
+	d = TRand.ar(1.6*i - detune, 1.7*i + detune, Impulse.ar(0)).round(0.25);
```

Comment:
because you are triggering only once (`Impulse.ar(0)`), you can use a `Rand` instead:

`d = Rand.ar(1.6 * i - detune, 1.7 * i + detune).round(0.25);`